spec/models/banished_user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe BanishedUser, type: :model do
+  describe "validations" do
+    describe "builtin validations" do
+      it { is_expected.to validate_uniqueness_of(:username).case_insensitive }
```

Comment:
I'm generally not a big fan of specs like this, since they basically just retest Rails. But you have them on other models, so I went for consistency.